Description: Disable integration tests
 due to fixtures absent in tests/examples/
Forwarded: not-needed
Author: Blair Noctis <ncts@debian.org>
Last-Update: 2025-03-25
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/integration_tests.rs
+++ b/tests/integration_tests.rs
@@ -1 +1,2 @@
+#![cfg(any())]
 use predicates::boolean::PredicateBooleanExt;
